tests/TextDiffTest.php

```diff
@@ -90,7 +90,7 @@ public function testAllFormatsCovered()
         sort($formats);
 
         $this->assertSame(
-            array('json', 'txt', 'xml'),
+            array('json', 'junit', 'txt', 'xml'),
```

Comment:
Which is a good thing, it means we found a little issue before it was merged :)
Please take a look at;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Report/XmlReporter.php#L71

I think your reporter needs something like that and the test should pass.